htdocs/product/fournisseurs.php

```diff
@@ -611,8 +611,9 @@ function on_change() {
         function update_price_from_multicurrency() {
             var multicurrency_price = $('input[name="multicurrency_price"]').val();
             var multicurrency_tx = $('input[name="multicurrency_tx"]').val();
-            $('input[name="price"]').val(multicurrency_price / multicurrency_tx);
-            $('input[name="disabled_price"]').val(multicurrency_price / multicurrency_tx);
+            var price=multicurrency_price.replace(",", ".") / multicurrency_tx.replace(",", ".");
```

Comment:
With some language, the decimal separator is someting else. Can you use instead the js method price2numjs on multicurrency_price and multicurrency_tx (it will replace the , or any other char depending on language to .)